cmd/gardener-admission-controller/main.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/gardener/gardener/cmd/gardener-admission-controller/app"
+	"github.com/gardener/gardener/cmd/utils"
+
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+)
+
+func main() {
+	ctx := utils.ContextFromStopChannel(signals.SetupSignalHandler())
+	command := app.NewCommandStartGardenerAdmissionController(ctx)
+
+	if err := command.Execute(); err != nil {
```

Comment:
Can you pass down the context via `ExecuteContext`? (You can then get it in `RunE` via `cmd.Context()`)
```suggestion
	command := app.NewCommandStartGardenerAdmissionController()

	if err := command.ExecuteContext(ctx); err != nil {
```